velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -803,6 +813,10 @@ size_t ParquetReader::numberOfRowGroups() const {
   return readerBase_->fileMetaData().row_groups.size();
 }
 
+common::CompressionKind ParquetReader::getCompressionKind() const {
```

Comment:
feels to me like we should take the row id and column chunk id as a parameter here. 

Agreed with @majetideepak 's point that in most cases they happen to be the same, but in case they don't, the `pageReader->getCompressionKind()` will be pretty unintuitive and likely not do what the API client would expect.

Reply:
This is fair. I think in a follow-up PR, we should add API to get RowGroupMetaData, ColumnChunkMetaData and make `getCompressionKind()` part of `ColumnChunkMetaData` where it technically belongs.